CMakeLists.txt

```diff
@@ -300,7 +300,7 @@ add_subdirectory(lonestar)
 if (ENABLE_DIST_GALOIS)
   add_subdirectory(lonestardist)
 endif()
-add_subdirectory(lonestarmine)
+#add_subdirectory(lonestarmine)
```

Comment:
oops, I'll put this back in in a commit before merging the pull request.

Note that having this with HETERO_GALOIS does break the build though, so this will need to be fixed for that.